plugins/org.locationtech.udig.catalog.wms/src/org/locationtech/udig/catalog/wms/preferences/WmsPreferenceInitializer.java

```diff
@@ -0,0 +1,24 @@
+package org.locationtech.udig.catalog.wms.preferences;
+
+import org.locationtech.udig.catalog.internal.wms.WmsPlugin;
+
+import org.eclipse.core.runtime.preferences.AbstractPreferenceInitializer;
+import org.eclipse.jface.preference.IPreferenceStore;
+
+
+/**
+ * 
+ * @author Nikolaos Pringouris <nprigour@gmail.com>
+ *
+ */
+public class WmsPreferenceInitializer extends AbstractPreferenceInitializer {
+
+    @Override
+    public void initializeDefaultPreferences() {
+        IPreferenceStore store = WmsPlugin.getDefault().getPreferenceStore();
+       
+        store.setDefault(WmsPreferencePage.WMS_RESPONSE_TIMEOUT, 30);

```

Comment:
can we define a constant with a semantic to describe what "30" means, e.g. DEFAULT_WMS_RESPONSE_TIMEOUT_IN_SECONDS:

`public static final int DEFAULT_WMS_RESPONSE_TIMEOUT_IN_SECONDS = 30;`

In addition, is it possible to Start uDig with another default (overwrite in configuration) and how?

@nprigour Could you add a screenshout to docs and improve WMS Preferences page (content of http://udig.github.io/docs/user/reference/Preferences.html)  or [Catalog Preferences](https://github.com/locationtech/udig-platform/blob/master/docs/user/en/reference/Catalog%20Preferences.rst)